packages/L10n/stories/L10n.stories.mdx

```diff
@@ -0,0 +1,16 @@
+import { Meta } from "@storybook/addon-docs/blocks";
+import { Description } from "@storybook/addon-docs/blocks";
```

Comment:
These imports can be combined.  
Too bad `eslint` doesn't properly check this in mdx files 😞